### ISSUE TITLE
[WPF] Outline of the drop down and icons on the top menu have insufficient contrast against their adjacent color

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#861]
The following AI-windows mode(s) have this issue:
- Dark mode
- High Contrast White mode
- High Contrast ```#1```
- High Contrast ```#2```

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![8 1](https://user-images.githubusercontent.com/22068434/88346485-4da9fe80-ccfd-11ea-93ee-ab909b0fc166.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by juchampa** [comment]
This is a WPF specific color contrast issue
